### PR TITLE
Update bundle_zh_CN.properties(ready to merge)

### PR DESCRIPTION
**Pull request by LaoHuaJiOfficial**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [ ] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by Anuken** [comment]
Has this been reviewed at all; can anyone confirm that this translation should be merged?

File: core/assets/bundles/bundle_zh_CN.properties [hunk 1]
```diff
@@ -994,46 +994,47 @@ stat.immunities = 免疫
...
+ability.energyfield = 能量场
+ability.energyfield.description = 对附近的敌人释放电击
+ability.energyfield.healdescription = 对附近的敌人释放电击，并治疗盟友
+ability.regen = 再生
+ability.regen.description = 随着时间的推移恢复自己的生命值
+ability.liquidregen = 液体吸收
+ability.liquidregen.description = 吸收液体以恢复自身
+ability.spawndeath = 死亡产生单位
+ability.spawndeath.description = 死亡时释放单位
+ability.liquidexplode = 液体爆炸
```
Comment (BlackDeluxeCat) [inline core/assets/bundles/bundle_zh_CN.properties:1023]: `ability.liquidexplode = Death Spillage`
"液体爆炸"是按字段名译的吗... 这里给出建议: 死亡溢液, 死亡产生液体
其实我很想写尸爆或者死亡爆浆之类的:D

File: core/assets/bundles/bundle_zh_CN.properties [hunk 2]
```diff
@@ -1176,14 +1177,14 @@ setting.position.name = 显示玩家坐标
 setting.mouseposition.name = 显示鼠标坐标
 setting.musicvol.name = 音乐音量
 setting.atmosphere.name = 显示行星大气层
-setting.drawlight.name = Draw Darkness/Lighting
+setting.drawlight.name = 绘制黑暗/光照
```
Comment (BlackDeluxeCat) [inline core/assets/bundles/bundle_zh_CN.properties:1180]: Darkness -> 阴影

File: core/assets/bundles/bundle_zh_CN.properties [hunk 3]
```diff
@@ -1129,7 +1130,7 @@ setting.backgroundpause.name = 游戏在后台时自动暂停
 setting.buildautopause.name = 自动暂停建造
 setting.doubletapmine.name = 双击采矿
 setting.commandmodehold.name = 长按保持指挥模式
-setting.distinctcontrolgroups.name = Limit One Control Group Per Unit 
+setting.distinctcontrolgroups.name = 每单位限制一个控制组
```
Comment (BlackDeluxeCat) [inline core/assets/bundles/bundle_zh_CN.properties:1133]: "控制组"是直译, 没有问题. 
但是这部分既然是RTS元素, 还是提个翻译建议: 控制组 -> 编队
后面所有含controlgroup的同上

File: core/assets/bundles/bundle_zh_CN.properties [hunk 4]
```diff
@@ -1221,23 +1222,23 @@ keybind.mouse_move.name = 单位跟随鼠标
...
+keybind.unit_stance_hold_fire.name = 单位姿态：停火
+keybind.unit_stance_pursue_target.name = 单位姿态：追逐目标
+keybind.unit_stance_patrol.name = 单位姿态：巡逻
+keybind.unit_stance_ram.name = 单位姿态：冲锋
+keybind.unit_command_move = 单位指挥：移动
+keybind.unit_command_repair = 单位指挥：修复
+keybind.unit_command_rebuild = 单位指挥：重建
+keybind.unit_command_assist = 单位指挥：协助
+keybind.unit_command_mine = 单位指挥：采矿
+keybind.unit_command_boost = 单位指挥：冲刺
```
Comment (BlackDeluxeCat) [inline core/assets/bundles/bundle_zh_CN.properties:1238]: boost问题, 同上
这里可以看到如果译为冲刺, 就容易和`keybind.unit_stance_ram.name = 单位姿态：冲锋`产生歧义了

File: core/assets/bundles/bundle_zh_CN.properties [hunk 5]
```diff
@@ -2408,7 +2410,7 @@ laccess.dead = 单位或建筑是否已被摧毁或者已失效
 laccess.controlled = 若单位的控制方是处理器，返回[accent]@ctrlProcessor[]\n若单位/建筑由玩家控制，返回[accent]@ctrlPlayer[]\n若单位在编队中，返回[accent]@ctrlFormation[]\n其他情况，返回0
 laccess.progress = 进度，0到1之间的数值。 \n返回工厂生产、 炮塔装填，或者建筑建造的进度
 laccess.speed = 单位的最高速度（格/秒）
-laccess.id = ID of a unit/block/item/liquid.\nThis is the inverse of the lookup operation.
+laccess.id = 单位/块/物品/液体的ID。\n这是查找操作的反向操作。
```
Comment (BlackDeluxeCat) [inline core/assets/bundles/bundle_zh_CN.properties:2413]: `laccess.id = 单位/块/物品/液体的ID。\n这是 Lookup 的反向操作。`
Lookup作为逻辑语言中的关键字, 不应译出.

**Review by BlackDeluxeCat — CHANGES_REQUESTED** [review]
看看你的

**Comment by LaoHuaJiOfficial** [comment]
@Anuken the translation is polished and able to merge now

**Pull request merged**